static/src/stylesheets/module/content-garnett/live-blog/_live-blog.head.scss

```diff
@@ -840,4 +881,30 @@ $pillars: (
     .tonal__standfirst ul>li::before {
         background: rgba(255, 255, 255, .4);
     }
+    .content__standfirst {
+        margin-left: -$gs-gutter / 2;
+        @include mq(mobileLandscape) {
+            margin-left: -$gs-gutter;
+        }
+        @include mq(phablet) {
+            margin-left: 0;
+        }
+    }
+}
+
+.js-sport-tabs {
```

Comment:
Usually not a good idea to use class names prefixed with `js-` in a stylesheet, by convention. Is there another class you could use? Not that we're going to change it, that is just for hygiene.

Reply:
Good to know, will update.